### DISCUSSION TITLE
Wireguard LXC / WGDashboard hangs

### DISCUSSION THREAD

**Discussion by KurtVanHove** [tteck/Proxmox#3578] [Help Needed]
Using the new helper install script of 2024-08-22 the install goes flawless.
However, when using WGDashboard, the application hangs after a while.
It starts eating RAM at that time. Console is also really slow.

![image](https://github.com/user-attachments/assets/4354722c-dc4b-4639-9bc4-d547ff63a5bb)

I experience this when adding peers (saving) but I've had this also before I was able to click Save.
New at Proxmox, complete newb at wireguard, so if you need more info, tell me where to look.

![image](https://github.com/user-attachments/assets/3ec2dd70-aae5-4a1b-bf22-72b2bcb11288)

**Comment by KurtVanHove** [comment]
Found the solution - bad configuration on my part (of WGDashboard). You can delete this request, since I'm not able to.
In the Configuration settings during the creation, right below the Listen Port, was "IP Address & Range".
I thought this was the client range, but this should've been the internal range.

**Reply by speedersen** [reply]
I'm an amateur when it comes to network setup, but I usually manage to set up various IT things anyway.

I experienced exactly the same problem as you and initially thought it was a bigger issue, like a memory leak or something similar.
Now it seems like I might be setting it up incorrectly.

My network is running on 192.168.43.0/24. What should I then write under peer in that case?

Any help is appreciated.

**Reply by speedersen** [reply]
Just a quick follow-up, I've solved my problem. I read up a bit on subnets and subnet masks, and also realized that I was missing `postup` and `postdown` in my config. It's working now, and without consuming memory.
